packages/template-retail-react-app/jest-setup.js

```diff
@@ -6,10 +6,11 @@
  */
 /* eslint-env jest */
 /* eslint-disable @typescript-eslint/no-var-requires */
-const path = require('path')
-const mockConfig = require(path.join(__dirname, 'config/mocks/default.js'))
+// fetch polyfill can be removed when node 16 is no longer supported
+require('cross-fetch/polyfill')
```

Comment:
I ran the tests in the retail react app 10 times. This one line change _significantly_ reduces the size of the log output:
* Total lines: 9308 --> 6140
* `console.error`s: 131 --> 109
* `console.warn`s: 178 --> **0**!

Reply:
Hooray for incremental improvement 🎉